Chargebee/Classes/Purchase/CBPurchaseManager.swift

```diff
@@ -19,7 +19,7 @@ public class CBPurchase: NSObject {
 
     private var restoredPurchasesCount = 0
     private var activeProduct: SKProduct?
-    var customerID: String = ""
+    var customerInfo: CBCustomer?
```

Comment:
Just `customer` should be good and keep it private?

Reply:
ok we can update as customer but if we keep it private then user won't have have right to pass the customer information so it should be public.